libs/server/Resp/RespCommand.cs

```diff
@@ -650,7 +650,10 @@ private RespCommand FastParseCommand(byte* ptr)
                     //[$8|SMEMBERS|] = 14 bytes = 8 (long) + 2 (ushort) + 2 bytes
                     if (*(long*)ptr == 5567941533359749156L && *(ushort*)(ptr + 8) == 17730 && *(ptr + 13) == 10)
                         return (RespCommand.Set, (byte)SetOperation.SMEMBERS);
-                    //[$4|SREM|] = 10 bytes = 8 (long) + 2 (ushort)
+                    //[$9\r\nSISM EMBER\r\n]
+                    if (*(long*)ptr == 5571877784987187492L && *(ushort*)(ptr + 8) == 19781 && *(ptr + 14) == 10)
```

Comment:
Not all of the input is being verified here. 

Try this:
//[$9|SISMEMBER|] = 15 bytes = 8 (long) + 4 (int) + 2 (ushort) + 1 byte
if (\*(long\*)ptr == 5571877784987187492L && \*(int\*)(ptr + 8) == 1161973061 && \*(ushort\*)(ptr + 12) == 3410 && \*(ptr + 14) == 10)

Reply:
done